kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/summary/AltoidaSummaryProcessor.kt

```diff
@@ -20,8 +20,7 @@ class AltoidaSummaryProcessor : StatelessCsvLineProcessor() {
             "YEARSOFEDUCATION",
             "GENDER",
             "CLASS",
-            "NMI",
-            "GROUNDTRUTH")
+            "NMI")
```

Comment:
@nivemaham I removed `GROUNDTRUTH` because it's not in the excel file. Is this fine?